auth-api/src/auth_api/services/membership.py

```diff
@@ -223,7 +224,7 @@ def update_membership(self, updated_fields, **kwargs):
 
         # Ensure that a member does not upgrade a member to ADMIN from COORDINATOR unless they are an ADMIN themselves
         if self._model.membership_type.code == COORDINATOR and updated_fields.get("membership_type", None) == ADMIN:
-            check_auth(org_id=self._model.org_id, one_of_roles=(ADMIN, STAFF))
+            check_auth(org_id=self._model.org_id, one_of_roles=(ADMIN))
```

Comment:
Doesn't sound like we need a change in functionality for this piece, so we can restore this to previous as discussed